README.md

```diff
@@ -43,6 +43,20 @@ deploycron(content="* * * * * echo hello > /tmp/hello")
 deploycron(content="* * * * * echo hello > /tmp/hello", override=True)
 ```
 
+and 
+
+```python
+def undeploycron_between(start_line, stop_line):
+```
+
+> Uninstall crontab parts between two lines (included).
+> If the start_line or the stop_line is not found into the installed crontab,
```

Comment:
suggest adding some introductions to the arguments, for example:
- start_line: the crontab *string* used to determine where to start from.
- stop_line: the crontab *string* used to determine where to stop at.